### ISSUE TITLE
NVDA is not reading the aria-selected status of list item

### ISSUE THREAD

**Issue by yugimamidi** [nvaccess/nvda#9061]
#### Have you tried any other versions of NVDA? No
[aria-select.txt](https://github.com/nvaccess/nvda/files/2674945/aria-select.txt)

**Comment by leonardder** [comment]
Please provide steps to reproduce as well as the browser(s) you tried this with.

**Comment by leonardder** [comment]
I've looked into the example you provided, and I don't think that this is actually a bug. NVDA does not report the selected state for selected items in a list, only the unselected state. NVDA also behaves like this in desktop applications, i.e. the list with desktop icons.

Unless you still have a reason to believe that NVDA is in error here, I'll have to close this.

**Comment by Adriani90** [comment]
@leonardder, @yugimamidi as I am not a developer, what should be spoken when aria selected is supported? Should be reported that the item is selected? Or should just expose the selected item in the list to NVDA when using browse mode?

**Comment by Adriani90** [comment]
After exploring further issues, I think we can close this as duplicate of #8587. Please comment on it if this was my mistake and we can reopen it. Otherwise let's continue the discussion on #8587.
